module/module.go

```diff
@@ -277,14 +359,62 @@ func readDeps(jsonDeps map[string][]string) []Dependency {
 	return deps
 }
 
+func readPackageInfo(
+	packageRoot string,
+	baseDirectory string,
+	targetGenDir string,
+	className string,
+	instanceDirectory string,
+	jsonConfig *JSONClassConfig,
+	dependencies []Dependency,
+) (*PackageInfo, error) {
+	qualifiedClassName := strings.Title(camelCase(className))
+	qualifiedInstanceName := strings.Title(camelCase(jsonConfig.Name))
+	defaultAlias := camelCase(strings.ToLower(qualifiedInstanceName)) +
+		qualifiedClassName
+
+	relativeGeneratedPath, err := filepath.Rel(baseDirectory, targetGenDir)
+	if err != nil {
+		return nil, errors.Wrapf(
+			err,
+			"Error computing generated import string for %s",
+			targetGenDir,
+		)
+	}
+
+	return &PackageInfo{
+		PackageName:           defaultAlias,
+		PackageAlias:          defaultAlias + "Static",
+		GeneratedPackageAlias: defaultAlias + "Generated",
+		PackagePath: filepath.Join(
+			packageRoot,
+			instanceDirectory,
+		),
+		ExportName: qualifiedInstanceName,
+		ExportType: qualifiedInstanceName + qualifiedClassName,
+		GeneratedPackagePath: filepath.Join(
+			packageRoot,
+			relativeGeneratedPath,
+			instanceDirectory,
+		),
+		IsExportGenerated: jsonConfig.IsExportGenerated == nil ||
```

Comment:
Should we do a `IsNotGenerated` boolean instead of having missing field default to `true`. That way `undefined` and `false` are "generated" and `true` is "generated"